blame.go

```diff
@@ -33,12 +37,6 @@ type BlameResult struct {
 // 2. Then build a graph with a node for every line in every file in
 // the history of the file.
 //
-// Each node (line) holds the commit where it was introduced or
-// last modified. To achieve that we use the FORWARD algorithm
-// described in Zimmermann, et al. "Mining Version Archives for
-// Co-changed Lines", in proceedings of the Mining Software
-// Repositories workshop, Shanghai, May 22-23, 2006.
-//
```

Comment:
Regarding to https://github.com/src-d/go-git/issues/231 (General questions) how the things are implemented is not documentation.

I will keep this comment inside the method.